.env.example

```diff
@@ -54,3 +54,5 @@ PHOENIX_API_VERSION=v1
 BLINK_URL=
 BLINK_USERNAME=
 BLINK_PASSWORD=
+
+SLACK_WEBHOOK_INTEGRATION_URL=
```

Comment:
I actually could just update the room in the Slack user interface but use the same URL! Updated this here since this is a new thing we'll need. Should I add this to Heroku now? 

After that's added, this seems like it's good to go! Unless I should test? But that might ruin the surprise/could be the surprise reveal?!

Reply:
It couldn't hurt to test one more time in `#team-bleed`, but then let's do it!!!

Reply:
@chloealee I think we only want the url to be set for production so tags from Thor and Staging don't hit `#badass_members` ? Unless we can use a different url that sends to #team-bleed for us to test stuff?

Reply:
@katiecrane ah so smart!! I'll remove those all those except for on Prod after I test on Thor!